translate.py

```diff
@@ -5,7 +5,7 @@
 
 
 def translator(string: str) -> str:
-    return string.rstrip('.') + ')'
+    return string.rstrip('.') +'.'+'.'+'.'+ ')'
```

Comment:
Why not 
```python
return string.rstrip('.') + '...)'
```
?

And also we have ellipsis symbol `…` instead of three dots `...`.